controller/workitemtype_blackbox_test.go

```diff
@@ -132,6 +133,14 @@ func (s *workItemTypeSuite) createWorkItemTypeAnimal() (http.ResponseWriter, *ap
 		Request: &http.Request{Host: "api.service.domain.org"},
 	}
 	spaceSelfURL := rest.AbsoluteURL(reqLong, app.SpaceHref(space.SystemSpace.String()))
+
+	categoriesData := []*app.GenericData{}
```

Comment:
please. make a private function for this. We repeat it several times in the code